From bb80ca87d3765d6a7d0a33fe703429ee79be71f7 Mon Sep 17 00:00:00 2001
From: Cathy Hu <cathy.hu@suse.com>
Date: Thu, 12 Jun 2025 13:55:42 +0200
Subject: [PATCH 1/3] Fix selinux policy for /usr/bin equivalency rules

fedora needs /usr/bin instead of /usr/sbin due to equivalency rules:
https://github.com/fedora-selinux/selinux-policy/commit/1be14f9b5a99a4eec7f9aba7fbb83bf8dde817f4

openSUSE has the fedora selinux policy as upstream, so also fixes it there
---
 selinux/drbd.fc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/selinux/drbd.fc b/selinux/drbd.fc
index 389d452369b0..6b5fcdd7da94 100644
--- a/selinux/drbd.fc
+++ b/selinux/drbd.fc
@@ -12,8 +12,8 @@
 
 /usr/lib/ocf/resource\.d/linbit/drbd	--	gen_context(system_u:object_r:drbd_exec_t,s0)
 
-/usr/sbin/drbdadm	--	gen_context(system_u:object_r:drbd_exec_t,s0)
-/usr/sbin/drbdsetup	--	gen_context(system_u:object_r:drbd_exec_t,s0)
+/usr/s?bin/drbdadm	--	gen_context(system_u:object_r:drbd_exec_t,s0)
+/usr/s?bin/drbdsetup	--	gen_context(system_u:object_r:drbd_exec_t,s0)
 
 /var/lib/drbd(/.*)?	gen_context(system_u:object_r:drbd_var_lib_t,s0)
 
-- 
2.43.0

